openpifpaf/show/painters.py

```diff
@@ -15,34 +15,6 @@
 LOG = logging.getLogger(__name__)
 
 
-class AnnotationPainter:
```

Comment:
Can you keep the location of the AnnotationPainter so that this diff doesn't go crazy?